app/helpers/duration_formatter_helper.rb

```diff
@@ -0,0 +1,6 @@
+module DurationFormatterHelper
+  def format_duration(minutes)
+    dur = ActiveSupport::Duration.build(minutes * 60)
+    Time.at(dur).utc.strftime("%kh %-Mm").strip
```

Comment:
For consistency, we should use the `Time.to_s` method with and use a formatter specified in the `config/time_formats.rb`. See here for [docs](https://api.rubyonrails.org/classes/Time.html#method-i-to_formatted_s)

Reply:
What's the call to `strip` for?

Reply:
The `%k` and `%-M` formats remove leading zeros but pad with spacing (which don't fit the design and is also harder to test). `.strip` removes leading (and trailing) whitespace.

Reply:
Done df89760

Reply:
Would using ""%-kh %-Mm" work? The `-` should prevent padding with space.

Reply:
Done 05087bc